pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -679,6 +686,12 @@ def cmd_connection_select(context, name, options):
     # Update the root context making this context the basis for future
     # commands in the current interactive session
     ContextObj.update_root_click_context(new_ctx)
+
+    # close any existing connection.
+    # TODO/KS: Validate that the first test is required.
```

Comment:
Given the check in ContextObj.is_connected (both the property in this PR and the method in the second PR), it is not needed.

Reply:
Modified in part 2 so I did not update this..

Reply:
ok